<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>700</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>700</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>90</y>
      <width>551</width>
      <height>456</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <property name="spacing">
      <number>32</number>
     </property>
     <item row="0" column="2">
      <widget class="TicButton" name="btn_0_2">
       <property name="styleSheet">
        <string notr="true">font-size:100px;</string>
       </property>
       <property name="text">
        <string>_</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="TicButton" name="btn_0_0">
       <property name="styleSheet">
        <string notr="true">font-size:100px;</string>
       </property>
       <property name="text">
        <string>_</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="TicButton" name="btn_0_1">
       <property name="styleSheet">
        <string notr="true">font-size:100px;</string>
       </property>
       <property name="text">
        <string>_</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="TicButton" name="btn_2_2">
       <property name="styleSheet">
        <string notr="true">font-size:100px;</string>
       </property>
       <property name="text">
        <string>_</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="TicButton" name="btn_1_2">
       <property name="styleSheet">
        <string notr="true">font-size:100px;</string>
       </property>
       <property name="text">
        <string>_</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="TicButton" name="btn_1_0">
       <property name="styleSheet">
        <string notr="true">font-size:100px;</string>
       </property>
       <property name="text">
        <string>_</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="TicButton" name="btn_2_1">
       <property name="styleSheet">
        <string notr="true">font-size:100px;</string>
       </property>
       <property name="text">
        <string>_</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="TicButton" name="btn_1_1">
       <property name="styleSheet">
        <string notr="true">font-size:100px;</string>
       </property>
       <property name="text">
        <string>_</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="TicButton" name="btn_2_0">
       <property name="styleSheet">
        <string notr="true">font-size:100px;</string>
       </property>
       <property name="text">
        <string>_</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="RestartGame">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>50</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Restart Game</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMode">
    <property name="title">
     <string>Mode</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionComputer"/>
    <addaction name="actionTwo_Player"/>
   </widget>
   <addaction name="menuMode"/>
  </widget>
  <action name="actionComputer">
   <property name="text">
    <string>Computer</string>
   </property>
  </action>
  <action name="actionTwo_Player">
   <property name="text">
    <string>Two Player</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TicButton</class>
   <extends>QPushButton</extends>
   <header location="global">ticbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
